_posts/2024-07-02-release.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Release 2024.27
+author: Priya
+date: 2024-07-02
+category: Jekyll
+layout: post
+---
+### Enhancement:
+Implemented restrictions on connectors for free users. Free plan users now have access to MSSQL, MYSQL, and POSTGRES connectors. All other connectors are classified as premium.
```

Comment:
This is more a configuration rather than enhancement. Need not mention this here.